tests/config/test_omegaconf_config.py

```diff
@@ -722,9 +722,7 @@ def test_ignore_hidden_keys(self, tmp_path):
         _write_yaml(tmp_path / _BASE_ENV / "catalog1.yml", {"k1": "v1", "_k2": "v2"})
         _write_yaml(tmp_path / _BASE_ENV / "catalog2.yml", {"k3": "v3", "_k2": "v4"})
 
-        conf = OmegaConfigLoader(
-            str(tmp_path), base_env=_BASE_ENV, default_run_env=_DEFAULT_RUN_ENV
-        )
+        conf = OmegaConfigLoader(str(tmp_path), base_env=_BASE_ENV)
         conf.default_run_env = ""
```

Comment:
This line isn't needed anymore right?

Reply:
you mean ` conf.default_run_env = ""`? I have removed it.